soc/nxp/imxrt/imxrt118x/soc.c

```diff
@@ -366,6 +384,14 @@ static ALWAYS_INLINE void clock_init(void)
 
 #endif /* CONFIG_CAN_MCUX_FLEXCAN */
 
+#if !(DT_NODE_HAS_COMPAT(DT_CHOSEN(zephyr_flash), nxp_imx_flexspi_nor)) &&              \
```

Comment:
This check is wrong- we care if the *parent* node has the `nxp,imx-flexspi-nor` compatible. See here for the fix I applied to the RT1170: https://github.com/zephyrproject-rtos/zephyr/pull/80017/commits/c95fbdc5bf1ddee331fe34f4c6f453e2d50f5bd1

Reply:
Updated. Thank you.